bindings/go/scip/memtest/low_mem_test.go

```diff
@@ -0,0 +1,59 @@
+package memtest
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"runtime"
+	"runtime/debug"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/proto"
+
+	"github.com/sourcegraph/scip/bindings/go/scip"
+)
+
+func TestLowMemoryParsing(t *testing.T) {
+	tmpFile, err := os.CreateTemp(t.TempDir(), "very-large-index.scip")
+	require.NoError(t, err)
+	defer os.RemoveAll(tmpFile.Name())
+
+	const textSize = 128 * 1024
+	const docCount = 1000
```

Comment:
Can we get a rough estimation in a comment of the resulting index size from these values? Tried to gauge it at-a-glance but the strings/repeat having an indeterminate number of characters fuzzed me up. Lower bound is good as well.

Reply:
Index size should be about `(textSize + δ) * docCount` where δ < 100 probably based on carrying some integers related to offsets and lengths. So this will end up being about 128 MB.

Reply:
Added a comment.